unittests/misc_tests.cpp

```diff
@@ -1128,6 +1128,91 @@ BOOST_AUTO_TEST_CASE(stable_priority_queue_test) {
   } FC_LOG_AND_RETHROW()
 }
 
+BOOST_AUTO_TEST_CASE(varint_unpack) {
```

Comment:
`libraries/fc` has its own unit tests and this should probably be there with other basic unit tests for varint encoding.

Reply:
sure will fix it.